assets/js/containers/Layout/Layout.js

```diff
@@ -1,32 +1,116 @@
 import React from 'react';
-import { Link } from "react-router-dom";
+import {Link} from "react-router-dom";
+
 
 import Background from '../../components/UI/Background/Background';
 
+const meniu =
+    <div className="hide-text">
+        <hr class="m-0"/>
+        <div className="m-0 p-2 text-left h6">
+            <a className=" text-dark" href="#services">My Profile</a>
+        </div>
+        <hr class="m-0"/>
+        <div className="m-0 p-2 text-left h6">
+            <a className=" text-dark" href="#contact">Reserve</a>
+        </div>
+        <hr class="m-0"/>
+        <div className="m-0 p-2 text-left h6">
+            <a className="text-dark" href="#contact">Watch Users</a>
+        </div>
+        <hr class="m-0"/>
+    </div>;
+
+const openMeniu = () => {
+    document.getElementById("mySidenav").style.width = "100%";
+};
+
+const closeMeniu = () => {
+    document.getElementById("mySidenav").style.width = "0%";
+};
+
 const layout = (props) => (
     <>
         <Background/>
-        <header>
-            {/* there should be header (in our case without any links) */}
-            <div style={{backgroundColor:'#f1a177', height:'4em', color:'white', fontWeight:'900', fontSize:'1.5em', boxShadow:' 0px 0px 10px 0px grey'}}>
-                
-                    <span style={{float:'left', marginLeft: '3em', marginTop:'1em'}}>NFQ parking</span>
-                    <span style={{float:'right', marginRight: '3em', marginTop:'1em'}}>Vardenis Pavardenis</span>
+        <div className="container-fluid">
```

Comment:
Try to create different component for the menu and add it to layout container. Also, links are not connected. Use react-router -dom links because there are routes already created.  exp. <Link to='/home' > Home </Link>